### ISSUE TITLE
How to evaluate the lora finetune checkpoint?

### ISSUE THREAD

**Issue by Lewislou** [AttentionX/InstructBLIP_PEFT#5]
Hi,

After finetuning, how to evaluate the checkpoint using testint set? I did not find any codes for loading lora weights.

Best,

**Comment by waitzkin** [comment]
In the codebase, the test set evaluation is conducted after the train/validation loop in the train function located in runner_base.py.

I believe this part is the most critical part.
https://github.com/AttentionX/InstructBLIP_PEFT/blob/main/lavis/runners/runner_base.py#L483

By loading the correct model into the "model" variable, I think you'll be able to evaluate benchmarks using the trained checkpoints.

Let me know if you have any further questions, and feel free to email me (sk0428@snu.ac.kr) if you need in-depth assistance.
